.ci.yaml

```diff
@@ -1108,6 +1108,20 @@ targets:
           {"dependency": "vs_build", "version": "version:vs2019"}
         ]
 
+  - name: Windows_arm64 windows-build_all_packages stable
+    recipe: packages/packages
+    timeout: 30
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
```

Comment:
We can remove this; feedback from infra recently is that we should err on the side of only adding things to the recipes CQ if they are substantially different than existing things that are there from a recipe standpoint.